microsoft-edge/devtools-guide-chromium/search/search-tool.md

```diff
@@ -68,13 +68,13 @@ The **Search** tool shows the list of matching resources, and highlights the cor
 
 By default, the **Search** tool isn't case-sensitive.  A search for a term matches occurrences of that term, regardless of lowercase or uppercase characters.
 
-To only find results that match a particular case (lowercase or uppercase characters), click the **Match Case** (`Aa`) button in the search toolbar.
+To only find results that match a particular case (lowercase or uppercase characters), click the **Match Case** (**Aa**) button in the search toolbar.
 
 
 <!-- ====================================================================== -->
 ## Search for regular expressions
 
-You can use regular expressions to find matching results.  To use a regular expression, click the **Use Regular Expression** (`.*`) button in the toolbar and enter a valid [JavaScript regular expression](https://developer.mozilla.org/docs/Web/JavaScript/Guide/Regular_Expressions) in the search input field.
+You can use regular expressions to find matching results.  To use a regular expression, click the **Use Regular Expression** (**.\***) button in the toolbar and enter a valid [JavaScript regular expression](https://developer.mozilla.org/docs/Web/JavaScript/Guide/Regular_Expressions) in the search input field.
```

Comment:
@mikehoffms was "\" in **.\*** supposed to escape an asterix?

Reply:
@dan-wesley - not sure the question.

gh rendered shows the UI and the formatting result:
https://github.com/mikehoffms/edge-developer/blob/user/mikehoffms/keys/microsoft-edge/devtools-guide-chromium/search/search-tool.md#search-for-regular-expressions

Reply:
was wondering about the escape in front of * .. but see it renders correctly. so, no issue.